website/shared/upgrade/index.mdx

```diff
@@ -0,0 +1,17 @@
+
+import UpgradeMetaClI from "./meta-cli.mdx";
+import UpgradePythonSDK from "./python-sdk.mdx";
+import UpgardeTsSDK from "./typescript-sdk.mdx";
+
+#
+
+## Upgrade your Metatype development environment
+
+### Meta CLI
+<UpgradeMetaClI />
+
+### Typescript SDK
+<UpgardeTsSDK />
+
+### Python SDK
+<UpgradePythonSDK />
```

Comment:
How about using tabs? Also since it will be used anywhere maybe we should remove any titles?